hc/api/models.py

```diff
@@ -114,7 +114,8 @@ def to_dict(self):
             "tags": self.tags,
             "timeout": int(self.timeout.total_seconds()),
             "grace": int(self.grace.total_seconds()),
-            "n_pings": self.n_pings
+            "n_pings": self.n_pings,
+            "status": self.status
```

Comment:
Yes, a cleaner option would be to have a separate flag for grace status. And we can add it when there's actually a need for it.